charts/tidb-lightning/templates/_helpers.tpl

```diff
@@ -22,6 +22,21 @@ config-file: |-
     {{- if .Values.config }}
 {{ .Values.config | indent 2 }}
     {{- end -}}
+    {{- if and .Values.tlsCluster .Values.tlsCluster.enabled }}
+  [security]
+  ca-path="/etc/lib/lightning-tls/ca.crt"
```

Comment:
For the other components, we mount the secrets to `/var/lib/...`, I think we can keep consistent with the others.

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/3598/commits/704b30b0f4c26e52ff41e1cf023b9ea1bc611ef1.